spec/features/trainees/edit_programme_details_spec.rb

```diff
@@ -43,6 +81,16 @@ def and_i_submit_the_form
     @programme_details_page.submit_button.click
   end
 
+  def and_i_visit_the_summary_page
+    @summary_page ||= PageObjects::Trainees::Summary.new
```

Comment:
You can get rid of all instances of `@summary_page ||= PageObjects::Trainees::Summary.new` and just use the attr reader `summary_page` which is declared in `spec/support/features/trainee_steps.rb`

Reply:
done https://github.com/DFE-Digital/register-trainee-teachers/pull/176/commits/7e11f2317b5e083113e8eca1b35f16704c43452b